src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -1090,34 +1090,34 @@ void WorldSession::HandleInspectHonorStatsOpcode(WorldPacket& recv_data)
     SendPacket(&data);
 }
 
-void WorldSession::HandleWorldTeleportOpcode(WorldPacket& recv_data)
+void WorldSession::HandleWorldTeleportOpcode(WorldPacket& message)
 {
-    uint32 time;
-    uint32 mapid;
-    float PositionX;
-    float PositionY;
-    float PositionZ;
-    float Orientation;
+    if (!AccountMgr::IsGMAccount(GetSecurity()))
+    {
+        SendNotification(LANG_YOU_NOT_HAVE_PERMISSION);
+        return;
+    }
+
+    uint32 requestTimeMs    = message.read<uint32>();
+    uint32 mapID            = message.read<uint32>();
+    uint64 player           = message.read<uint64>();
+    Position position       = message.read<Position>();
 
-    recv_data >> time;                                      // time in m.sec.
-    recv_data >> mapid;
-    recv_data >> PositionX;
-    recv_data >> PositionY;
-    recv_data >> PositionZ;
-    recv_data >> Orientation;                               // o (3.141593 = 180 degrees)
+    //  IF THE PLAYER GUID IS NULL IT MEANS
+    //  THE USER ISSUING THE COMMAND IS THE TARGET
+    Player* playerPtr = player ? ObjectAccessor::FindPlayer((ObjectGuid)player) : GetPlayer();
 
-    if (GetPlayer()->IsInFlight())
+    if (!playerPtr)
     {
-        LOG_DEBUG("network", "Player '{}' ({}) in flight, ignore worldport command.", GetPlayer()->GetName(), GetPlayer()->GetGUID().ToString());
+        SendNotification("Player not found");   //  DO NOT TRANSLATE
```

Comment:
What's special about this string specifically that it should not be translated?

Reply:
Also, you should prefix it with "WORLDPORT: ...."

Reply:
In modern internal builds, the client has an extra message handler that displays the console message _Player not found_.
The message does not seem to exist as of the LK alpha/beta builds and no handler can be found for it in patch 3.3.5a.

It is safe to assume the functionality was not present in patch 3.3.5a.
However, we can imagine that Blizzard would still display the error message, as many commands could in theory use it. Add to that the fact that Blizzard sends notifications when cheats are toggled on and off and the theory starts to make sense.

Reply:
I appreciate the attention to detail but mimicking their internal builds is not something we aim for when it comes to something that adds no value to the project, like a single error string not localised

Reply:
We try to mimic gameplay as much as possible. But admin/gm features etc sometimes can be fun to try and mimic, but errors strings like these are no point in mimicing